docs/resources/chatbot_slack_channel_configuration.md

```diff
@@ -10,6 +9,26 @@ description: |-
 
 Resource schema for AWS::Chatbot::SlackChannelConfiguration.
 
+## Example Usage
```

Comment:
you need to sync your local repo and then re-run `make docs` to update the .md file